rest-mvc/src/main/java/com/intuit/tank/rest/mvc/rest/controllers/AgentController.java

```diff
@@ -80,10 +81,10 @@ public ResponseEntity<org.springframework.core.io.Resource> getSupportFiles() th
     }
 
     @RequestMapping(value = "/ready", method = RequestMethod.POST, consumes = { MediaType.APPLICATION_JSON_VALUE })
-    @Operation(description = "Registers an agent instance to a job and sets it's status to ready to start", summary = "Set an agent instance status to ready to start")
+    @Operation(description = "Registers an agent instance to a job and sets it's status to ready to start", summary = "Set an agent instance status to ready to start", hidden = true)
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "Successfully set agent to ready"),
-            @ApiResponse(responseCode = "400", description = "Bad request", content = @Content)
+            @ApiResponse(responseCode = "400", description = "Could not register agent", content = @Content)
```

Comment:
@Zakaria-Kofiro Can we update these 400 descriptions in the PR to reflect that user made a bad request? Otherwise from the description seems like it can be misinterpreted as a server-side error

Reply:
Updated